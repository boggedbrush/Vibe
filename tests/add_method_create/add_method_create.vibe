# VibeSpec: 1.0
patch_type: add_method
file: hello.py
class: Greeter
--- code: |
def greet(self):
    print("Greeter says hi!")

