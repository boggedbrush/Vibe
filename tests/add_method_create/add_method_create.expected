# hello.py
def greet(name):
    print(f"Hello, {name}!")

class Greeter:
    def __init__(self, name):
        print(f'Hello {name}')

    def greet(self):
        print("Greeter says hi!")

